arangosh/Import/ImportHelper.cpp

```diff
@@ -827,7 +921,26 @@ void ImportHelper::addLastField(char const* field, size_t fieldLength,
     return;
   }
 
-  addField(field, fieldLength, row, column, escaped);
+  addField(field, fieldLength, row, column++, escaped);
+
+  if (!_attrsToValues.empty()) {
+    for (auto& [key, value] : _attrsToValues) {
+      if (row == 0) {
+        addField(key.c_str(), key.size(), row, column, escaped);
+      } else {
+        std::string attrsToMerge;
+        std::for_each(value.begin(), value.end(), [=, &attrsToMerge](std::pair<std::string, bool> const& attr) {
+          if (!attr.second && _fieldsLookUpTable.find(attr.first) != _fieldsLookUpTable.end()) {
+            attrsToMerge += _fieldsLookUpTable.at(attr.first);
```

Comment:
This will do 2 lookups for the value in the unordered_map if there is an entry for `column`, once for `find()` and once for `at()`. We could get away with just a single lookup.
```suggestion
          if (!attr.second && _fieldsLookUpTable.find(attr.first) != _fieldsLookUpTable.end()) {
            attrsToMerge += _fieldsLookUpTable.at(attr.first);
```